Include/cpython/dictobject.h

```diff
@@ -37,7 +37,8 @@ typedef struct {
 
 PyAPI_FUNC(PyObject *) _PyDict_GetItem_KnownHash(PyObject *mp, PyObject *key,
                                                  Py_hash_t hash);
-PyAPI_FUNC(PyObject *) _PyDict_GetItemStringWithError(PyObject *, const char *);
+// PyDict_GetItemStringRef() can be used instead
+Py_DEPRECATED(3.14) PyAPI_FUNC(PyObject *) _PyDict_GetItemStringWithError(PyObject *, const char *);
```

Comment:
Maybe 3.13?

Reply:
I don't like adding new compiler warnings after beta1. There is no urgency to remove it.